index.html

```diff
@@ -145,29 +148,151 @@ <h2 class="content-home-heading" id="content-home-heading">The best destinations
+  <aside class="profile-bg">
+    <div class="profile" >
+      <button class="profile-user"><img src="img/user.png" alt=""></button>
+      <div class="profile-p-b">
+        <h3>Sign up and get 20% off your first booking</h3>
+        <button class="profile-sing-button">Sign up</button>
+      </div>
+      <button class="profile-clouse"><img src="img/Vector.svg" alt=""></button>
+    </div>
+  </aside>
 
   <aside class="mob-profile-bg">
     <div class="mob-profile" >
-    <button class="mob-profile-user"><img class="mob-profile-user-i" src="img/Vector.png" alt=""></button>
-    <img class="tripHouseMob-img" src="img/tripHouseMob.svg" alt="tripHouseMob">
-    <div class="mob-profile-h-a">
-      <h3 class="mob-profile-h" >Sign up in the app and get 20% off your first booking</h3>
-      <a class="mob-profile-a">Install</a>
-    </div>
+      <button class="mob-profile-user"><img class="mob-profile-user-i" src="img/Vector.png" alt=""></button>
+      <img class="tripHouseMob-img" src="img/tripHouseMob.svg" alt="tripHouseMob">
+      <div class="mob-profile-h-a">
+        <h3 class="mob-profile-h" >Sign up in the app and get 20% off your first booking</h3>
+        <a class="mob-profile-a">Install</a>
+      </div>
     </div>
   </aside>
 </section>
+
+<!-- Guests reviews -->
+<section>
+  <div class="guests-bg">
+    <h2 class="content-home-heading-des">Guests reviews</h2>
+    <h2 class=" content-home-heading-mob">Reviews</h2>
+    <div class="reviews">
+      <div class="reviews-width" id="reviews-width">
+        <div class="content-reviews" id="content-reviews">
+          <div class="content-reviews-img">
+            <figure class="hotel">
+              <img src="img/hotel.png" alt="hotel">
+              <figcaption> <p>From 250 USD</p> </figcaption>
+            </figure>
+            <div class="content-reviwes-img-fig">
+              <div class="location">
+                <div class="location-p">
+                  <p>Ubud Bali Resort&SPA</p>
+                  <p>Bali, Indonesia</p>
+                </div>
+                <div class="estimation">
+                  <img src="img/score.svg" alt="">
+                  <p> 1324 reviews </p>
+                </div>
+              </div>
+
+              <div class="users-reviews">
+                <img class="users-reviews-img" src="img/AccountCircle 1.png" alt="">
+                <div class="users-data" id="users-data">
+                  <h4>Hanna Ivanova</h4>
+                  <figure class="users-country">
+                    <img src="img/Ukraine.png" alt="">
+                    <figcaption>Ukraine</figcaption>
+                  </figure>
+                </div>
+              </div>
+              <p class="users-country-reviews" id="users-country-reviews">Great location, really pleasant and clean rooms, but the thing that makes this such a good place to stay are the staff. All of the people are incredibly helpful and generous with their time and advice.</p>
```

Comment:
Again there is wrong styles for text. Why? **You should take font styles them from figma!**
**font-size: 18px;**

Reply:
add font-size:18px